drivers/hmis/app/models/hmis/hud/validators/custom_service_validator.rb

```diff
@@ -0,0 +1,26 @@
+class Hmis::Hud::Validators::CustomServiceValidator < Hmis::Hud::Validators::BaseValidator
+  IGNORED = [
+    :ExportID,
+    :ServicesID,
+    :DateCreated,
+    :DateUpdated,
+    :RecordType,
+    :TypeProvided,
+    :OtherTypeProvided,
+    :MovingOnOtherType,
+    :SubTypeProvided,
+    :FAAmount,
+    :ReferralOutcome,
+  ].freeze
+
+  def configuration
+    Hmis::Hud::Service.hmis_configuration(version: '2022').except(*IGNORED)
+  end
+
+  def validate(record)
+    super(record) do
+      # Ensure that HUD services are not saved to the CustomServices table
+      record.errors.add :custom_service_type, :invalid, message: 'is a HUD service' if record.service_type.hud_service?
```

Comment:
There's nothing the user could do about this. It would only happen if we have a bug. I just wanted to make sure there will be an error to catch it.

I'm not sure what you mean by catching it at the creation of the service type. The service types can and do include HUD types, so that's OK.

Reply:
Sounds good.  I was thinking if you tried to add a custom service with an identical name as a HUD service, maybe that would be a problem.